### PR TITLE
fix: fix translucent screen issue with ion-content

### PR DESCRIPTION
**Pull request by bistaastha**
Adds an `ion-content` container tag.

## Summary by CodeRabbit

- **New Features**
	- Introduced `<ion-content>` wrappers in various pages to enhance layout and scrolling behavior.
	- Improved organization of forms and error messages across password reset and verification pages.

- **Bug Fixes**
	- Retained and reorganized error messages for better visibility and consistency.

- **Style**
	- Adjusted padding properties in CSS for improved spacing in the password reset and invited user components. 

- **Refactor**
	- Enhanced HTML structure and readability without changing functionality.
	- Updated password fields to be hidden by default for improved user experience.

### CODE REVIEW

File: src/app/auth/new-password/new-password.page.html [hunk 1]
```diff
@@ -1,106 +1,108 @@
-<div class="new-password">
-  <div>
-    <ion-icon
-      class="fy-icon new-password__back-icon"
-      src="../../../assets/svg/arrow-tail-left.svg"
-      (click)="redirectToSignIn()"
-    ></ion-icon>
+<ion-content scroll-y="false">
```
Comment (bistaastha) [inline src/app/auth/new-password/new-password.page.html:1]: indentation changes

**Review by Chethan-Fyle — APPROVED** [review]

File: src/app/post-verification/invited-user/invited-user.page.html [hunk 2]
```diff
@@ -8,132 +1,141 @@
...
+    <img
+      class="invited-user-offline__image"
+      alt="offline-zer-state"
+      src="../../../assets/images/zero-states/offline.svg"
+    />
+    <div class="invited-user-offline__title">Looks like you are offline</div>
+    <div class="invited-user-offline__sub-title">
+      You need to be connected to Internet to be able to login to the app. Please check if your Internet connection is
+      working.
+    </div>
```
[the remark this answers is not in the document]
Comment (bistaastha) [reply src/app/post-verification/invited-user/invited-user.page.html:139]: not needed

File: src/app/auth/pending-verification/pending-verification.page.html [hunk 3]
```diff
@@ -1,88 +1,93 @@
...
             [src]="'/assets/svg/arrow-left.svg'"
             slot="icon-only"
           ></ion-icon>
           <span class="pending-verification__cta-text">Back to Sign In</span>
         </div>
-      </ion-button>
+      </div>
+    </div>
+    <div class="pending-verification__send-invite" *ngIf="isInvitationLinkSent">
+      <div class="pending-verification__content-container" *ngIf="isInvitationLinkSent">
```
[the remark this answers is not in the document]
Comment (bistaastha) [reply src/app/auth/pending-verification/pending-verification.page.html:57]: Fixed in a follow up

File: src/app/auth/reset-password/reset-password.page.html [hunk 4]
```diff
@@ -1,88 +1,94 @@
...
+        </section>
+      </div>
+      <div class="forgot-password__cta">
+        <ion-button
+          (click)="onGotoSignInClick()"
+          class="btn-primary"
+          fill="clear"
+          aria-label="Navigate back to sign in page"
+          role="button"
+        >
```
[the remark this answers is not in the document]
Comment (bistaastha) [reply src/app/auth/reset-password/reset-password.page.html:81]: mat tooltip is not useable

**Pull request merged**